cypress/support/commands.js

```diff
@@ -33,7 +33,7 @@ Cypress.Commands.add('anotherLoginUser', (email, password) => {
     cy.get('[name="email"]').type(email)
       .get('[name="password"]').type(password)
   });
-  cy.get('[name="submit"]').click()
+  cy.get('[name="submit"]').first().click()
```

Comment:
Why do you need to call first on those elements? Can’t you try to identify them with both name AND value? There is a danger with your approach if there are new elements added later.

Reply:
The reason behind it was that both have the type and name submit, it's the login and the cancel button below it. Changed name of second btn now.